usecases/traverser/explorer.go

```diff
@@ -156,6 +157,10 @@ func (e *Explorer) getClassKeywordBased(ctx context.Context, params dto.GetParam
 
 	res, err := e.search.ClassSearch(ctx, params)
 	if err != nil {
+		var e inverted.MissingIndexError
+		if errors.As(err, &e) {
```

Comment:
We are returning an error anyway. I think it is better to return the whole error. 
I would try this instead
`fmt.Errorf("explorer: list class: search: %w", err)`

Reply:
This change was introduced to address the issue https://github.com/weaviate/weaviate/issues/2974.
Error was simplified and made human-friendly only for one particular case: index is not configured for property used in gql query, which may be quite common case for beginner users. All other, less probable, errors will have still full message returned.